billboard.py

```diff
@@ -314,7 +324,43 @@ def getMinistatsCellValue(fieldName, ifNoValue=None):
             self.entries.append(entry)
 
     def _parseNewStylePage(self, soup):
-        dateElement = soup.select_one("button.date-selector__button.button--link")
+        artists = []
```

Comment:
Can you move this whole block into a helper function?

Reply:
done